accounting/receivables/customer_invoices/payment_terms.rst

```diff
@@ -72,25 +76,33 @@ Payment Terms can be defined with the **Payment Terms** field on:
   | To set specific payment terms on a bill. This is mostly useful when you need to manage vendor
     terms with several installments. Otherwise, setting the *Due Date* is enough.
 
+.. note::
+   You can also define a **Due Date** manually. If Payment Terms are already defined, empty the
+   field so you can select a date.
+
+Payment Terms can be defined with the **Payment Terms** field on:
+
+- | **Contacts**
+  | To set specific payment terms automatically on new sales orders, invoices, and bills of a
+    contact. This can be modified in the contact’s *Form View*, under the *Sales & Purchase* tab.
+
 Journal Entries
 ===============
+
 Invoices with specific Payment Terms generate different *Journal Entries*, with one *Journal Item*
-for each different *Due Date* computed.
+for each *Due Date* computed.
 
 This makes for easier *Follow-ups* and *Reconciliation* since Odoo takes each due date into account,
-rather than just the balance due date.
+rather than just the balance due date. It also helps to get an accurate *Aged Receivable report*.
 
-In the following example, an invoice of $1000 has been issued with the following payment terms:
-30% of the invoice is due on the day of issuance of the invoice, and the balance is due at the end
-of the following month.
-
-.. image:: media/payment_terms_journal_entry.png
+.. image:: payment_terms/journal-entry.png
    :align: center
-   :alt: Example of an invoice with specific Payment Terms. The amount debited on the Account
-         Receivable is split in several Journal Items.
+   :alt: The amount debited on the Account Receivable is split into two Journal Items with distinct
+         Due Dates
 
-The $1000 debited on the Account Receivable is split into two distinct *Journal Items*. Both of
-them have their own **Due Date**.
+In this example, an invoice of $1000 has been issued with the following payment terms: *30% of the
+invoice is due on the day of issuance of the invoice, and the balance is due at the end of the
+following month.*
```

Comment:
Can you say, "30% is due on the day of issuance, and the balance is due at the end of the following month." I think "of the invoice" is implied.

Reply:
true!